.storybook/design-system-docs-decorator.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { StoryContext } from '@storybook/react/*';
+import { StoryContext } from '@storybook/react';
```

Comment:
type check complained about this

Reply:
Thanks. I don't understand why auto-import adds `/*`.